src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -405,6 +405,7 @@ static void create_stub_global_symbol(
   new_symbol.base_name = symbol_basename;
   new_symbol.type = symbol_type;
   new_symbol.type.set(ID_C_class, class_id);
+  new_symbol.type.set(ID_C_access, ID_public); // Guessed.
```

Comment:
Shouldn't this be ID_private to make us defensive?

Reply:
Neither is better than the other; both invite some form of accidental incorrectness. I've added a comment to that effect.

Reply:
I'm not certain I follow - if we say it is private, and its public, we use reflection to access it, no harm no fowl. If we do the opposite, we produce tests that don't compile. I suppose your concern is to do with the inheriting the component (if private we will fall through and try and make a new static field in the parent class)? I suppose in this case at least we can, in test generation find out if it is a stub and if it is defensively use reflection.